serial_views.py

```diff
@@ -0,0 +1,153 @@
+        No geometry is included.
+        All related tables are joined and the values are included as property within the GEOJSON.
+
+
+        :return:
+        """
+
+        features = []
+
+        for record in Serializer.session.query(oep_modles.classes['Series'],
+                                    oep_modles.classes['Timespan'],
+                                    oep_modles.classes['Variable']).join(oep_modles.classes['Timespan'])\
+                                                                   .join(oep_modles.classes['Variable']).limit(1000):
+
+            # Collection all Columns to be included from tables timespan and values
+            # ToDo: maybe serialize the following on another session
+            # record.Series.timespan.segments not included "list to long"
+            # record.Series.values not included "list to long"
+            # ToDo: How to handel the DateTimeObj so it is Json Serializeable
+            # timespan_collection = {"Start": record.Series.timespan.start.strftime('%b %d %Y %I:%M%p'),
+            #                        "Stop": record.Series.timespan.stop.strftime('%b %d %Y %I:%M%p'),
+            #                        "Resolution": record.Series.timespan.resolution}
+
+            netcdf = record.Series.variable.netcdf_attributes
+            variables_collection = {"Name": record.Series.variable.name}
+            variables_collection["NetCDF"] = netcdf
+
+
+            propertys = {"SeriesID": record.Series.id, "values": record.Series.values, "height": record.Series.height}
+            # propertys.update(timespan_collection)
+            propertys.update(variables_collection)
+            feature = Feature(id=record.Series.id, properties=propertys)
+            features.append(feature)
+
+        return HttpResponse(dumps(FeatureCollection(features)), content_type="application/json")
+
+        # def wseries_get_single_point(self, request):
+        #     """
+        #     Return the data for the closest weather-point for a given position
+        #     as GeoJSON.
+        #     The given position is provided as HTTP POST/GET method.
+        #
+        #     request: is the current mouse position passed from client
+        #     :return:
+        #     """
+        #     if request.method is 'POST':
+        #         position = request.POST.get('latlng')
+        #
+        #         for record in Serializer.session.query(oep_modles.classes['Series'],
+        #                                                oep_modles.classes['Timespan'],
+        #                                                oep_modles.classes['Variable'])\
+        #                                                 .join(oep_modles.classes['Timespan']) \
+        #                                                 .join(oep_modles.classes['Variable']).limit(1000):
+        #
+        #
+        #             return HttpResponse(geojsondata, content_type="application/json")
+
+    def pp_list_geometry_view(self):
```

Comment:
already added the example function`s to keep in mind what needs to be done.

Reply:
"the example function" --> "empty functions"